test/unit/tracks/text-tracks.test.js

```diff
@@ -175,37 +166,38 @@ test('update texttrack buttons on removetrack or addtrack', function() {
     };
   };
 
-  tag = document.createElement('video');
-  track = document.createElement('track');
+  const tag = document.createElement('video');
+  const track = document.createElement('track');
+
   track.kind = 'captions';
   track.label = 'en';
   track.language = 'English';
   track.src = '#en.vtt';
   tag.appendChild(track);
-  track = document.createElement('track');
```

Comment:
the issue is that we want to create a new track here. With the current change, we end up with one track being created, have various attributes set and appended to tag in line 184/176 but then below this line it ends up re-using the same track element which we don't want.